src/components/EntityInfo.vue

```diff
@@ -273,6 +285,16 @@ export default class EntityInfo extends Mixins(CommonMixin, EnumMixin) {
 
 #entity-info {
   background: $BCgovInputBG;
+
+  &.hover {
+    background: $app-bg-lt-blue;
```

Comment:
Special UX request: change background on this entire component when the user hovers over the View and Change Company Information button.